lib/ansible/plugins/filter/core.py

```diff
@@ -283,11 +283,19 @@ def mandatory(a):
 
 def combine(*terms, **kwargs):
     recursive = kwargs.get('recursive', False)
-    if len(kwargs) > 1 or (len(kwargs) == 1 and 'recursive' not in kwargs):
-        raise AnsibleFilterError("'recursive' is the only valid keyword argument")
+    reverse = kwargs.get('reverse', False)
+
+    arg_error = "'recursive' and 'reverse' is the only valid keyword arguments"
+
+    for key in kwargs.keys():
+        if not (key == 'recursive' or key == 'reverse'):
```

Comment:
why not `if key not in ('recursive', 'reverse'):`? that's also easier to extend in the future.

Reply:
True!